backend/wordsList/LetterValues/dutch.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
```

Comment:
You might have difficulties with some german letters you might want find different tags for them